src/CMakeLists.txt

```diff
@@ -151,6 +151,9 @@ if(LINUX)
   list(APPEND EXTRALIBS ${LIB_RESOLV})
 endif(LINUX)
 
+# Embedded Library
+option(WITH_EMBEDDED "build the embedded ceph daemon library" ON)
```

Comment:
the argument for making it the default is that the code will not rot. I'm going to be adding unit tests to ensure the functionality does not break as we make changes in the code base. If we turn it off it will never get tested and will eventually rot. the increased build time is minimal from what I can tell.